library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -245,6 +248,7 @@ public enum DefaultIdManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GafferPopGraph.class);
     private static final String GET_ALL_DEBUG_MSG = "Requested a GetAllElements, results will be truncated to: {}.";
+    private static final Pattern TSTV_REGEX =  Pattern.compile("^tstv:(?<type>[A-Za-z0-9-]*)\\|(?<stype>[A-Za-z0-9-]*)\\|(?<val>[A-Za-z0-9-]*)$");
```

Comment:
just to check that TSTV's can't contain a pipe in the type/subtype/value ?

Reply:
Updated the regex match to allow seeding like `g.V("t:thetype|st:thesubstype|v:theval")` this should hopefully avoid majority of conflicts